fall23/deepface/detectfaces.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.                    # Use extract_faces function without specifying the backend, letting DeepFace use its default
</pre><p>Would be good to compare different backend models if the default is not accurate enough.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-atfal-mafkoda-missing-children/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>